Mail/Views/NoMailboxView.swift

```diff
@@ -24,25 +25,17 @@ struct NoMailboxView: View {
     @Environment(\.openURL) private var openURL
 
     @State private var isShowingLoginView = false
-    let slide = Slide(
-        id: 1,
-        backgroundImage: MailResourcesAsset.onboardingBackground3.swiftUIImage,
-        title: MailResourcesStrings.Localizable.noMailboxTitle,
-        description: MailResourcesStrings.Localizable.noMailboxDescription,
-        asset: MailResourcesAsset.noMailbox.swiftUIImage
-    )
 
-    var body: some View {
-        VStack(spacing: 0) {
-            SlideView(slide: slide)
-                .overlay(alignment: .top) {
-                    MailResourcesAsset.logoText.swiftUIImage
-                        .resizable()
-                        .scaledToFit()
-                        .frame(height: UIConstants.onboardingLogoHeight)
-                        .padding(.top, UIPadding.onBoardingLogoTop)
-                }
+    private let slide = Slide(backgroundImage: MailResourcesAsset.onboardingBackground3.image,
+                              backgroundImageTintColor: UserDefaults.shared.accentColor.secondary.color,
+                              content: .illustration(MailResourcesAsset.noMailbox.image),
+                              bottomViewController: UIHostingController(rootView: OnboardingTextView(
```

Comment:
Would it be possible to have a slide init that accepts a View?
kDrive is the only app that will provide actual UIKit. Mail and SwissTransfer will both provide View embedded in UIHostingController.